.github/workflows/CI.yml

```diff
@@ -17,6 +17,10 @@ jobs:
         uses: actions/setup-node@v2.1.2
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Debug version.
+        run: npm --version
+      - name: Update npm to version 7.
+        run: npm install -g npm@latest
```

Comment:
Yes, it's needed. However, please do move it to `npm ci` as `npm i -g npm@7 && npm ci`.